staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcdmetrics
```

Comment:
why this package name? Without good reasons this should be called `metrics`

Reply:
@xiang90 suggests we should use that and know it's a etcd metric ,see above Xiang's review

Reply:
@sttts

We should avoid generic package name unless the package itself is a generic one. This can improve readability, avoid import naming conflicts and confusion. For example, in go standard pkg, it is httputil not util, httptrace not trace, bufin instead of io, etc..

Reply:
I don't care too much about this. But we have pkg=dir as a convention everywhere in kubernetes. Why should we make an exception here?